README.md

```diff
@@ -66,8 +66,8 @@ For example, if you want to use PostgreSQL, add to your `mix.exs` file:
 
 ```elixir
 defp deps do
-  [{:postgrex, ">= 0.0.0"},
-   {:ecto, "~> 0.14.3"}]
+  [{:postgrex, ">= 0.9.1"},
+   {:ecto, "~> 1.0.0"}]
```

Comment:
ok,
link to understand the diference : http://elixir-lang.org/docs/stable/elixir/Version.html